sway-lsp/src/capabilities/hover/mod.rs

```diff
@@ -126,11 +123,9 @@ fn hover_format(
     session: Arc<Session>,
     engines: &Engines,
     token: &Token,
-    ident: &Ident,
+    token_name: &str,
```

Comment:
Nice, but would `ident_name` be more accurate here?

Reply:
Yep that makes more sense, I change it now.